main.go

```diff
@@ -107,7 +111,7 @@ func init() {
 
 func newSubnetManager() (subnet.Manager, error) {
 	if opts.kubeSubnetMgr {
-		return kube.NewSubnetManager()
+		return kube.NewSubnetManager(opts.kubeApiUrl, opts.kubeConfigFile)
```

Comment:
I don't think it's needed since this is a dev/test focused feature, it's more important to keep the code clean and small here.